tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/SelectSeqAsFold.scala

```diff
@@ -0,0 +1,87 @@
+package at.forsyte.apalache.tla.pp
```

Comment:
I'd suggest adding some unit tests for this.

Reply:
This is somewhat related to discussions we've had with Igor on this topic, i.e. whether to do unit tests or integration tests. The short version is, that it's very tedious to unit test processing passes, because you'd have to write tests that check if your transformed value has a very specific syntactic shape. OTOH, the tests in `SelectSeqTests` implicitly require that the transformation produces the result we expect, but are significantly easier to write. Unless there's some particular property of the transformation you'd like to verify in unit testing, integration testing suffices, IMO.

Reply:
Ok. Make sense.